src/shared_utilities/SpaceVaryingInputs.jl

```diff
@@ -137,29 +137,21 @@ function SpaceVaryingInput(
 end
 
 """
-    SpaceVaryingInput(data::PDS, varname::N; space::S) 
-    where {PDS <: FileReader.PrescribedDataStatic, N <: String, S <: ClimaCore.Spaces.SpectralElement2D}
-  
+    SpaceVaryingInput(data_handler::DataHandler)
+    SpaceVaryingInput(file_path::AbstractString, varname::AbstractString, target_space::Spaces.AbstractSpace;)
+
 Returns the parameter field to be used in the model; appropriate when
 a parameter is defined on the surface of the Earth.
 
-Pass the data as a `FileReader.PrescribedDataStatic` object
-as well as the variable name of the variable in the data file, and the ClimaCore space
-of the model simulation.
-
 Returns a ClimaCore.Fields.Field of scalars; analogous to the 1D case which also
 returns a ClimaCore.Fields.Field of scalars.
 """
-function SpaceVaryingInput(
-    data::PDS,
-    varname::N,
-    space::S,
-) where {
-    PDS <: FileReader.PrescribedDataStatic,
-    N <: String,
-    S <: ClimaCore.Spaces.SpectralElementSpace2D,
-}
-    return FileReader.get_data_at_date(data, space, varname)
+function SpaceVaryingInput(data_handler::DataHandler)
+    return snapshot(data_handler)
```

Comment:
repeating myself here, but I think this could be `regridded_snapshot` or `regridded_field` or something else for clarity

Reply:
especially here it is a little confusing (outside of the data handler module)

Reply:
Renamed to `regridded_snapshot`